### PR TITLE
build-in: add sin,cos,tan build-in function

### PR DESCRIPTION
**Pull request by willxm**

### CODE REVIEW

File: expression/builtin_math.go [hunk 1]
```diff
@@ -776,7 +776,21 @@ type builtinCotSig struct {
...
+	arg := args[0]
+	if arg.IsNull() {
+		return d, nil
+	}
+	sc := b.ctx.GetSessionVars().StmtCtx
+	x, err := arg.ToFloat64(sc)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	d.SetFloat64(math.Cos(x)
```
Comment (coocood) [inline expression/builtin_math.go]: The function is `COT`, not `COS`.

File: expression/builtin_math_test.go [hunk 2]
```diff
@@ -351,3 +353,75 @@ func (s *testEvaluatorSuite) TestSqrt(c *C) {
...
+		ret interface{}
+	}{
+		{math.Pi / 2, float64(1)},
+		{math.Pi, float64(1.2246467991473515e-16)},
+		{math.Pi / 3, float64(0.8660254037844387)},
+		{float64(1), float64(0.8414709848078965)},
+	}
+
+	for _, test := range testCase {
+		fc := funcs[ast.Sqrt]
```
Comment (coocood) [inline expression/builtin_math_test.go]: This will get a function of `SQRT`.

**Comment by willxm** [comment]
Sorry, I copy that from SQRT function, but i forget to run go test. I am
immediately modified.

**Comment by ngaut** [comment]
Please fix CI.

**Comment by XuHuaiyu** [comment]
@willxm 
If possible, please check and register the builtin function you decide to implement in [issue](https://github.com/pingcap/tidb/issues/2863) to let others know, thank you!

**Comment by coocood** [comment]
@willxm 
Close this PR for now, will reopen if you have any update.

**Pull request closed without merging**